<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title_lable">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Форма регистрации</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="user_bio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>291</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="placeholderText">
     <string>Введите биографию</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usuer_fio">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="placeholderText">
     <string>Введите ФИО</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>552</x>
      <y>100</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>среднее</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>неполное высшее</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>высшее</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="title_edq">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Образование</string>
    </property>
   </widget>
   <widget class="QLabel" name="title_date">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>170</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Дата рождения</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="user_birth">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>170</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="button_save_user">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>420</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сохранить пользователя</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clean">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>420</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>сбросить пароль</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
